dotCMS/src/test/java/com/dotcms/util/pagination/SiteViewPaginatorUnitTest.java

```diff
@@ -0,0 +1,211 @@
+package com.dotcms.util.pagination;
+
+import static org.mockito.Matchers.any;
+import static org.mockito.Matchers.anyBoolean;
+import static org.mockito.Matchers.anyInt;
+import static org.mockito.Matchers.anyString;
+import static org.mockito.Matchers.eq;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import com.dotcms.rest.api.v1.secret.SiteViewPaginator;
+import com.dotcms.rest.api.v1.secret.view.SiteView;
+import com.dotmarketing.beans.Host;
+import com.dotmarketing.common.model.ContentletSearch;
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.exception.DotSecurityException;
+import com.dotmarketing.portlets.contentlet.business.ContentletAPI;
+import com.dotmarketing.portlets.contentlet.business.HostAPI;
+import com.dotmarketing.util.PaginatedArrayList;
+import com.dotmarketing.util.UUIDUtil;
+import com.google.common.collect.Ordering;
+import com.liferay.portal.model.User;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Random;
+import java.util.Set;
+import java.util.function.Supplier;
+import java.util.stream.Collectors;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class SiteViewPaginatorUnitTest {
+
+    private static final char[] alphabet = "ABCDEFGHIJKLMNOPQRSTUVWXYZ".toCharArray();
+
+    @Test
+    public void Test_Get_Items_Page_Size()
+            throws DotSecurityException, DotDataException {
+
+        final int max = alphabet.length - 1;
+        final User user = mockAdminUser();
+        final List<String> allSites = mockAllSitesIdentifiers(max);
+        final Set<String> sitesWithIntegrations = mockSitesWithIntegrations(allSites, 10);
+        //System.out.println("Sites with integrations: ");
+        sitesWithIntegrations.forEach(System.out::println);
+        final HostAPI hostAPI = mock(HostAPI.class);
+        long time = System.currentTimeMillis();
+        int i = 0;
+        //System.out.println("Site names: ");
+        for(final String identifier:allSites){
+            final Host host;
+            if(Host.SYSTEM_HOST.equals(identifier)){
+                host = mockSite(identifier, "System Host");
+                when(hostAPI.find(eq(identifier),any(User.class), anyBoolean())).thenReturn(host);
+            } else {
+                final String name = alphabet[i++] + "" +  time;
+                System.out.println(name);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [System.out.println is used](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=4960082)